nodejs/eks/examples/README.md

```diff
@@ -5,7 +5,6 @@ clusters.
 
 1. [Default Cluster and a Custom Cluster](./cluster)
 1. [Cluster with NodeGroups](./nodegroup)
-1. [Cluster with Workers in Private Subnets](./private-workers)
```

Comment:
Why is this being removed?

Reply:
I updated the commit message and OP. Thoughts?